<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NorseVPN</class>
 <widget class="QMainWindow" name="NorseVPN">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>NorseVPN</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../doc/img/logo/NorseVPN_app.svg</normaloff>../doc/img/logo/NorseVPN_app.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="cb_connected">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>736</x>
      <y>10</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>checked you are currently connected to a vpn</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/white_link.svg</normaloff>icons/white_link.svg</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>560</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_debug">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>560</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_config">
    <property name="geometry">
     <rect>
      <x>752</x>
      <y>40</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/white_settings.svg</normaloff>icons/white_settings.svg</iconset>
    </property>
   </widget>
   <widget class="QTreeWidget" name="tree_location">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>561</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Location</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
    <property name="markdown">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_shuffle">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/white_shuffle.svg</normaloff>icons/white_shuffle.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_version">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Version v0.00.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
